homeassistant/components/recorder/statistics.py

```diff
@@ -0,0 +1,138 @@
+"""Statistics helper."""
+from __future__ import annotations
+
+from collections import defaultdict
+from datetime import datetime, timedelta
+from itertools import groupby
+import logging
+from typing import TYPE_CHECKING
+
+from sqlalchemy import bindparam
+from sqlalchemy.ext import baked
+
+import homeassistant.util.dt as dt_util
+
+from .const import DOMAIN
+from .models import Statistics, process_timestamp_to_utc_isoformat
+from .util import execute, retryable_database_job, session_scope
+
+if TYPE_CHECKING:
+    from . import Recorder
+
+QUERY_STATISTICS = [
+    Statistics.statistic_id,
+    Statistics.start,
+    Statistics.mean,
+    Statistics.min,
+    Statistics.max,
+]
+
+STATISTICS_BAKERY = "statistics_bakery"
```

Comment:
Since this is used to key `hass.data`, make the key start with the domain of the intgration.
```suggestion
STATISTICS_BAKERY = "recorder_statistics_bakery"
```